### PR TITLE
Portfolio Website-MobileFirst

### PR DESCRIPTION
**Pull request by evansnyamekye**
## Portfolio Website  🚀  
-I used HTML5 to build the entire portfolio page structure 🔧 .
-I used CSS to style and layout all the web pages contained in the mobile-first portfolio website ⚙️.
-Using CSS, I styled the required font, color, size, and spacing of the portfolio webpage 🧑‍💻 .
-I used Flexbox to properly align items on the portfolio webpage 🔲 .
As required by this project, I implemented button interactions to create an amazing portfolio website that will display nicely on all devices. 
Thank you 😀.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <i class='bx bx-menu'></i>
+          </div>
+        </nav>
+    
+        <!-- start of banner -->
+        <section class="banner">
+          <div class="container">
+            <div class="banner-text">
+              <h1>I’m Nyamekye. <br>
+                Glad to see you!</h1>
```
Comment (joskalenda) [inline index.html:38]: - [ ] Please fix this design below, we are expecting the `YOU!` to move to the red box position
- [ ] As for the bending line, please adjust your border radius at the bottom left position to match the original design

**Review by joskalenda — CHANGES_REQUESTED** [review]

**Review by joskalenda — CHANGES_REQUESTED** [review]
Hi @evansnyamekye , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

> You have submitted a project, that has not microverse readme [template](https://github.com/microverseinc/readme-template), please make sure you update your repo with a professional readme and request another review



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

-----
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._ 🟢

**Review by wayungi — COMMENTED** [review]
Hi @evansnyamekye , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project that is invalid. Please add  readme file using the [Micovrese template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by medaminedev66** [comment]
Hello @evansnyamekye  👋,

_This is an additional review as a result of the second opinion request._

I confirm that your README is descriptive and fulfills the requirements. Please go ahead and request a review through your dashboard.

Keep rocking 🚀!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                </ul>
+                <i class='bx bx-menu'></i>
+            </div>
+        </nav>
+    </header>
+    <!-- End of header container -->
+
+    <!-- start of Main -->
+    <main class="banner">
+        <div class="container">
```
Comment (V-Blaze) [inline index.html]: - - You have done great, your project looks like the Figma design. :+1: But there are some changes you need to make before moving to the next milestone. :smile: 
- [ ] Following the best practices for the use of semantic tags, this section should be wrapped in a `section` tag. the `section tag` represents a generic standalone section of the document, so I'll suggest you use it here. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. :smile: :wave:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @evansnyamekye  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and well-documented code✔
-  Descriptive pull request title👋 
-  UI looks great✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @evansnyamekye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**